pkg/cmd/roachtest/tests/s3_clone_backup_restore.go

```diff
@@ -24,48 +24,76 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
+type s3CloneSecureOption int
+
+const (
+	s3ClonePlain      s3CloneSecureOption = iota // Use HTTP
+	s3CloneRequireTLS                            // Use HTTPS, but skip certification verification.
```

Comment:
uber nit: you could rename `s3CloneRequireTLS` to `s3CloneTLSWithSkipVerify` to suggest this is non-default, and then `s3CloneVerifyTLS` to `s3CloneTLS`.